src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -774,15 +775,15 @@ Checkstyle ends with 5 errors.
 
     <section name="Customize token types in Javadoc Checks">
       <p>
-        Java checks controlled by method <a href="writingchecks.html#Understanding_token_sets">
+        Java checks are controlled by methods <a href="writingchecks.html#Understanding_token_sets">
           setTokens(), getDefaultTokens(), getAccessibleTokens(), getRequiredTokens(). </a>
-        JavaDoc checks use the same model plus extra 4 methods for Javadoc tokens.
-        As Java AST and Javadoc AST are not binded.
-        It is highly recommended for Javadoc checks do not use customization of java tokens and
-        expect to be executed only on javadoc tokens.
+        JavaDoc checks use the same model plus 4 additional methods for Javadoc tokens.
+        As Java AST and Javadoc AST are not binded,
+        it is highly recommended for Javadoc checks to not use customization of java tokens and
+        except to be executed only on javadoc tokens.
       </p>
       <p>
-        There are four methods in AbstractJavadocCheck class to control the processed
+        There are four methods in the AbstractJavadocCheck class to control the processed
```

Comment:
not required

Reply:
"methods in class AbstractJavadocCheck"?

Reply:
Not much difference.. anything will do